templates/layout-internal.html

```diff
@@ -18,6 +18,7 @@
       {% if request.user.is_authenticated %}
          | <a href="{% url "my-image-list" %}">My Uploads</a>
          | <a href="#">My Light Curves</a>
```

Comment:
You can delete this line - it was meant to be a placeholder.  And you can name yours "My Light Curves"

Reply:
Done